.buildkite/pipeline.yml

```
@@ -1,7 +1,7 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-  - &bash_cache automattic/bash-cache#2.12.0
+  - automattic/a8c-ci-toolkit#2.14.0
```

Version [2.15.0](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/tag/2.15.0) has been released in the meantime.

```suggestion
  - automattic/a8c-ci-toolkit#2.15.0
```